pyvrp/cpp/ProblemData.h

```diff
@@ -82,22 +98,29 @@ class ProblemData
      * @param servDurs     Service durations.
      * @param distMat      Distance matrix.
      */
-    ProblemData(std::vector<std::pair<int, int>> const &coords,
-                std::vector<int> const &demands,
-                size_t numVehicles,
-                size_t vehicleCap,
-                std::vector<std::pair<int, int>> const &timeWindows,
-                std::vector<int> const &servDurs,
-                std::vector<std::vector<int>> const &distMat);
+    ProblemData(
+        std::vector<std::pair<int, int>> const &coords,
+        std::vector<int> const &demands,
+        size_t numVehicles,
+        size_t vehicleCap,
+        std::vector<std::pair<duration_type, duration_type>> const &timeWindows,
+        std::vector<duration_type> const &servDurs,
+        std::vector<std::vector<duration_type>> const &distMat);
 };
 
 ProblemData::Client const &ProblemData::client(size_t client) const
 {
     return clients_[client];
 }
 
-int ProblemData::dist(size_t first, size_t second) const
+distance_type ProblemData::dist(size_t first, size_t second) const
```

Comment:
Maybe a stupid question and not related to this change but can we `inline` this?  Same for some of the other methods in this class like `client` which get called a lot?

Reply:
These are already inlined - you're looking at the implementation; `inline` is part of the declaration above (in the class).